src/pages/checkOutReport.js

```diff
@@ -0,0 +1,99 @@
+import { userAuthorization, removeActiveClass, hideAnimation, showAnimation, findParticipant, convertISODateTime, checkOutParticipant } from "./../shared.js"
+import { checkInTemplate } from './checkIn.js';
+import { homeNavBar, reportSideNavBar, unAuthorizedUser} from '../navbar.js';
+import fieldToConceptIdMapping from "../fieldToConceptIdMapping.js";
+
+
+export const checkOutReportTemplate = (auth, route) => {
+    auth.onAuthStateChanged(async user => {
+        if(user){
+            const response = await userAuthorization(route, user.displayName ? user.displayName : user.email);
+            if ( response.isBiospecimenUser === false ) {
```

Comment:
i noticed that we had the same code in `dashboard.js`. Should we create a common function in `shared.js`

```
if ( response.isBiospecimenUser === false ) {
                document.getElementById("contentBody").innerHTML = "Authorization failed you lack permissions to use this dashboard!";
                document.getElementById("navbarNavAltMarkup").innerHTML = unAuthorizedUser();
                return;
            }
```

Reply:
Hmm Im not aware of it. I'll turn into a function